src/docs/python-dependencies.mdx

```diff
@@ -21,10 +21,10 @@ So here are the rules we worked with so far:
 In order to avoid ever breaking master, you can:
 
 1. Open PR in getsentry to update requirements
-2. Open PR in sentry with the same branch name, put [`#sync-getsentry`](/workflow/) in PR description
+2. Open PR in sentry with the same branch name, put [`#sync-getsentry`](/workflow/) in the PR's first comment
```

Comment:
I'm not sure about this. This still leaves it a bit ambiguous, I would read this as "add another comment after creating the PR". 
What about clarifying what description means ("not the title, but the text you add when creating the PR"?) And if we want to be extra clear, we could also add a hint that the tag can be added later, but for the change to be picked up new changes have to be pushed to the branch.

Thinking about this now, someone could also update the bot to support GitHub labels. If the 'sync-getsentry' label is there => bot syncs. This will hopefully remove the description vs comment vs title ambiguity for good.